demo/src/screens/componentScreens/CardsScreen.tsx

```diff
@@ -125,6 +124,7 @@ export default class CardsScreen extends Component<CardsScreenProps, CardsScreen
     return (
       <Card style={{marginBottom: 10}} onPress={() => {}}>
         <Card.Section
+          bg-white
```

Comment:
Don't we want to have Card and Card.Section bg color white by default?

Reply:
Good question, I'm not sure.
@ethanshar, should we add `defaultProps` for `Card` and `CardSection` with `backgroundColor: Colors.white`?

Reply:
Yes, backgroundColor should be white by default

Reply:
Fixed